pages/reference/supervisor/bandwidth-reduction.md

```diff
@@ -4,52 +4,41 @@ title: Reduce bandwidth usage
 
 # Reduce bandwidth usage
 
-In order to provide users with essential information about their devices, the {{ $names.company.lower }} Supervisor on the device regularly keeps the API informed about the device's state. Upon receiving a target device state which differs from the current device state, the Supervisor applies changes, such as downloading new application updates or setting environment variables. While a quick response to actions is great during development, it also leads to of increased data usage.
+balenaOS and the device Supervisor provide convenient features that make it easy to manage a device remotely, like forwarding container logs, hardware metrics reporting, and Cloudlink for on-demand external access. However, these features consume network bandwidth, and typically cellular connectivity is charged by the megabyte. To allow you to trade off between cost and features, you may configure their use individually.
```

Comment:
I know you didn't want any comments on content, but shouldn't this have engine too? Or engine takes up no bandwidth?

Reply:
I struggled some to be comprehensive with these categories. The engine isn't involved here. It just runs what the Supervisor has provided for it.

Reply:
Confusion on my part; nothing further to do here.